pkg/builder/kaniko/kaniko.go

```diff
@@ -133,10 +133,7 @@ func (k *Kaniko) containerLogs(ctx context.Context, pod *v1.Pod) (string, error)
 }
 
 func (k *Kaniko) prepareJob(ctx context.Context, b *builder.BuilderOptions) (*batchv1.Job, error) {
-	jobName, err := names.NewRandomK8(kanikoJobNamePrefix)
-	if err != nil {
-		return nil, ErrGeneratingUUID.Wrap(err)
-	}
+	jobName := fmt.Sprintf("%s-%d", kanikoJobNamePrefix, time.Now().UnixNano())
```

Comment:
This is still random from the perspective that the user is not defining the name.

The intention behind the issue is that all names should be explicitly set by the user so that they have knowledge of them in order to reference and update things.

Reply:
it does not matter for kaniko as it creates a temp job to build and does not affect the long running tests.